src/routes/formsgSiteCreation.ts

```diff
@@ -75,16 +77,18 @@ export class FormsgRouter {
         await this.sendCreateError(requesterEmail, repoName, submissionId, err)
         return res.sendStatus(200)
       }
-      const foundUser = await this.usersService.findByEmail(requesterEmail)
-      if (!foundUser) {
+      const foundRequester = await this.usersService.findByEmail(requesterEmail)
```

Comment:
Nit: Could we rename this to `foundIsomerRequester` so we don't confuse this person with a `agencyFoundRequester`

Reply:
Fixed in https://github.com/isomerpages/isomercms-backend/pull/679/commits/f6cdb2b61f8f92ed453bf8782a2acae660fc95ad!